test/exercism/use_cases/attempt_test.rb

```diff
@@ -149,9 +152,12 @@ def test_previous_submission_after_superseding
   end
 
   def test_previous_submission_with_new_language_sandwich
-    Attempt.new(user, 'CODE 1', 'two/two.rb').save
-    Attempt.new(user, 'CODE 2', 'two/two.py').save
-    attempt = Attempt.new(user, Iteration.new({'ruby/two/two.rb' => 'CODE'}, {track: 'ruby', slug: 'two'})).save
+    Attempt.new(user, Iteration.new({'two/two.rb' => 'CODE 1'}, 
+    {track: 'ruby', slug: 'two'})).save
+    Attempt.new(user, Iteration.new({'two/two.py' => 'CODE 2'}, 
+    {track: 'python', slug: 'two'})).save
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.